src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -129,10 +131,15 @@ public JDA getJDA()
         return api;
     }
 
+    /**
+     * Returns the {@link MessageReference} for this Message. This will be null if this Message has no reference.
+     * @return The message reference.
```

Comment:
```suggestion
     *
     * @return The message reference.
```

Reply:
You don't need to add documentation to implementations of an interface that already has docs